app/detect_kubelet_arg.go

```diff
@@ -43,63 +47,81 @@ const (
       # no kubelet process found, echo nothing
     }
     get_flexvolume_path
-  `
-	DetectFlexVolumeMaxPolls = 120
+  	`
```

Comment:
You can abstract the script to be "get_kubelet_args" with a parameter name, e.g. "volume-plugin-dir" for Flexvolume and "root-dir" for root dir. The path reported in the `get_flexvolume_path` when cannot find the directory is only the default value, so it can be filled in outside the script, which is the same for `root-dir` as well.